community/server/src/main/java/org/neo4j/server/rest/web/RestfulGraphDatabase.java

```diff
@@ -552,9 +552,9 @@ public Response getNodesWithLabelAndProperty( @PathParam("label") String labelNa
 
     @GET
     @Path( PATH_LABELS )
-    public Response getAllLabels( )
+    public Response getAllLabels( @QueryParam( "in_use" ) @DefaultValue( "false" ) boolean inUse )
```

Comment:
I can see why the default value of the `in_use` flag is `false`: you want to preserve existing behaviour by default. However, I don't think that existing behaviour was actually useful for anyone. We only added this API specifically for the browser, and there is zero value there knowing about labels that were _once_ used. I suggest making the default value `true` so that new people coming to this API just get the answer they were expecting, and don't have to learn about and understand this parameter.